view/frontend/web/js/view/payment/method-renderer/omise-offline-conveniencestore-method.js

```diff
@@ -66,117 +66,20 @@ define(
                     }
                 };
             },
-
             /**
              * Initiate observable fields
              *
              * @return this
              */
             initObservable: function () {
                 this._super()
-                    .observe([
-                        'convenienceStorePhoneNumber',
-                        'convenienceStoreEmail',
-                        'convenienceStoreCustomersName'
-                    ]);
-                return this;
-            },
-
-            /**
-             * Checks if sandbox is turned on
-             *
-             * @return {boolean}
-             */
-            isSandboxOn: function () {
-                return window.checkoutConfig.isOmiseSandboxOn;
-            },
-
-            /**
-             * Get total amount of an order
-             *
-             * @return {integer}
-             */
-            getTotal: function () {
-                return + window.checkoutConfig.totalsData.grand_total;
-            },
-
-            /**
-             * Format Price
-             * 
-             * @param {float} amount - Amount to be formatted
-             * @return {string}
-             */
-            getFormattedAmount: function (amount) {
-                return priceUtils.formatPrice(amount, quote.getPriceFormat());
+                .observe([
+                    'convenienceStorePhoneNumber',
```

Comment:
There is repeating of these names. Feels like something that should be stored in maybe a const array that we should then iterate over